portality/scripts/article_cleanup_sync.py

```diff
@@ -0,0 +1,137 @@
+"""
+For each article in the DOAJ index:
+    * Checks that it has a corresponding journal, deletes it otherwise (if -d set)
```

Comment:
I can't foresee us wanting to keep orphaned articles, but for flexibility I would put it in. However, I would default to deleting if -w is specified.

It is going to be a little tricky to manage the interaction of the 2 flags and a potential source of bugs, so I'm wondering if it really is worth introducing. Don't put in the effort to introduce it for now - let's talk about it a bit tomorrow, and then it can be introduced if needed.